.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: release
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip poetry
+        poetry config virtualenvs.create false
+        poetry lock
+        poetry install
+
+    - name: Check codestyle
+      run: pre-commit run --all
+
+    - name: Build package
+      run: poetry build
+
+    - name: Release to PyPI
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+      run: twine upload --verbose dist/* || echo 'Version exists'
```

Comment:
I don't see twine in dependencies, is this installed somewhere?

Reply:
Right. Forgot that bit. Need to add it as a dev-dependency.